ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -283,6 +283,8 @@ def cdfPreamble(self, jobconfig):
         if self.globus_rsl:
             cdfDict['globus_rsl'] = self.globus_rsl
 
+        cdfDict['Requirements'] =  self.requirements.convert()
```

Comment:
The capital `R` in seems a bit odd when comparing to the rest of the `cdfDict`. Is it correct?

Reply:
Yes I agree it looks odd. I tried this with setting `OpSys` to `Linux` or a random string. The former ran, whilst the second was stuck in idle because the scheduler couldn't match the requirements so I think it works.

Looking around I have seen examples with all of these (`executable`, `universe` etc) with lower case and other examples with upper case so I think it may be case-insensitive.